tests/main/interfaces-cli/task.yaml

```diff
@@ -2,7 +2,7 @@ summary: Check the interfaces command
 
 environment:
     SNAP_NAME: network-consumer
-    SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+    SNAP_FILE: network-consumer_1.0_all.snap
```

Comment:
There are a bunch of similar ones that can probably be fixed in a similar way (if you want, it seems not too terrible to just write it explicit here actually).